.github/workflows/ci-2.x.yml

```diff
@@ -91,7 +92,7 @@ jobs:
         run: SYMFONY_REQUIRE="^4.4" composer update --no-progress --ansi --prefer-stable
 
       - name: Update project dependencies (5.*)
-        if: matrix.symfony == '5.*'
+        if: matrix.symfony == '5.*' || matrix.symfony == '6.*'
```

Comment:
I think you should keep 5.x and 6.x separate in two steps with something like this for sf5.x

```yaml
      - name: Update project dependencies (5.*)
        if: matrix.symfony == '5.*'
         run: SYMFONY_REQUIRE="^5.4" composer update --no-progress --ansi --prefer-stable
```